src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,197 @@
+
+package duke;
+
+import duke.tasks.Deadline;
+import duke.tasks.Events;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+import duke.tasks.Todo;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasklist;
+    private Ui ui;
+
+
+    private static String DIRECTORY = "./data";
+    private static String PATH = "./data/duke.txt";
```

Comment:
I like how constants are all uppercased. Perpaps doing "private static final " will be better?
```suggestion
    private static final String DIRECTORY = "./data";
    private static final String PATH = "./data/duke.txt";
```